ReactAndroid/src/main/java/com/facebook/react/views/text/ReactBaseTextShadowNode.java

```diff
@@ -462,7 +462,7 @@ public void setFontSize(float fontSize) {
     markUpdated();
   }
 
-  @ReactProp(name = ViewProps.COLOR)
+  @ReactProp(name = ViewProps.COLOR, customType = "Color")
```

Comment:
I wonder if there is a way to avoid this and check in the ViewManagersPropertyCache if the annotation name is color instead. Would that help?

Reply:
Checking the 'customType' attribute is easier because it is consistent across all the color props (except for this single exception that I fixed).   If I had to check the name, I'd have to check ViewProps.COLOR, ViewProps.BACKGROUND_COLOR, "borderColor", "overlayColor", "tintColor", "endFillColor", and on and on.

Reply:
Ah, so the other callsites of this already use `customType = 'color'`, it was just missing from this one?

Reply:
Correct